python-avd/pyavd/_cv/client/configlet.py

```diff
@@ -157,20 +165,68 @@ async def set_configlet_containers(
                     configlet_ids=RepeatedString(values=configlet_ids),
                     query=query,
                     child_assignment_ids=RepeatedString(values=child_assignment_ids),
-                    match_policy=ASSIGNMENT_MATCH_POLICY_MAP.get(match_policy),
+                    match_policy=ASSIGNMENT_MATCH_POLICY_MAP.get(match_policy or "match_all"),
                 )
                 for container_id, display_name, description, configlet_ids, query, child_assignment_ids, match_policy in containers
             ],
         )
         client = ConfigletAssignmentConfigServiceStub(self._channel)
         try:
-            responses = client.set_some(request, metadata=self._metadata, timeout=timeout + len(request.values) * 0.5)
+            responses = client.set_some(request, metadata=self._metadata, timeout=timeout)
             assignment_keys = [response.key async for response in responses]
         except Exception as e:
             raise get_cv_client_exception(e, f"Workspace ID '{workspace_id}', Containers '{containers}'") or e
 
         return assignment_keys
 
+    @LimitCvVersion(max_ver="2024.1.99")
```

Comment:
does this version exist?

Reply:
I will change max_ver to below_ver so I can use an existing version number instead of this.

Reply:
I cannot change this. The idea of 2024.1.99 means "every version below 2024.2.0" but since I cannot predict which 2024.1.x" version will be the last, I cannot set this.
Should we change the logic to use regex matching instead? It would be a bit ugly with CVaaS in there.

Reply:
lets aim at simplicity and add a comment on top explaining the why for future us